### ISSUE TITLE
proposing a community group "health check" / roll call

### ISSUE THREAD

**Issue by parispittman** [kubernetes/steering#153]
We have many community groups right now and this is a good feedback loop with chairs and steering. This is similar to the Apache PMC style roll call. 

### Proposed Solution

process:
- chair fills out questionnaire
- if steering has more questions, they invite chair to meeting (maybe May's Chair and Tech Leads meetings are this?)

**Why this is good:**
- Engaging community group operational leadership more with steering and community as a whole; steering can help advocate with more info
- Active, engaged, focused Chairs
- it's a part of our charter

### Open Questions

- [x] do we (steering) discuss any of this process (ie scheduling members for review, scheduling) during any of our public calls or is this a private process? all on the mailing list? etc
  - private process until further notice; will iterate once we do a round
- [x] are the sig chair/tl follow up meetings recorded? no
- [x] is this a large enough change that we follow the four week policy laid out in changes.md? no
- [x] what is the timeline for 2020?  
working group organizers will be the only ones for 2020 at this time. will PR this in a separate follow up once https://github.com/kubernetes/community/pull/4687 is merged

### Next Steps
- [x] Discussion (see steering meeting feb 3)
- [x] put together example questions / topics to explore (derek, timsc, paris)
- [x] (not dependent on completion of the last bullet) circulate this @ the chairs/tl meeting on feb 11 and 13
- [x] create survey
- [x] create the framework of a procedure and process https://github.com/kubernetes/community/pull/4687/files
- [x] edit all relevant docs: gov.md, sig-gov.md, wg-gov.md, ug-gov.md, annual-report.md https://github.com/kubernetes/community/pull/4687
- [x] start with working groups - follow up with a separate issue

### Other Considerations, Notes, or References
- https://github.com/kubernetes/community/issues/4289
- https://github.com/kubernetes/community/issues/3753
- https://www.apache.org/dev/pmc.html#roll-call (edited)

**Comment by parispittman** [comment]
questions that might be worth exploring:

working groups:
- [ ] how is the group doing on its mission?
- [ ] roadmap/timeframe until completion?
- [ ] learnings so far?
- [ ] is the working group active? healthy? contributors from multiple companies and/or end user companies?
- [ ] everything in your readme accurate? posting meetings on youtube?

sigs:
- [ ] what's it like being a chair for this group? do you want to continue or find a replacement? is there a succession plan in place? are there opportunities for others to step up in this role? 
- [ ] is your README accurate? subprojects listed in sigs.yaml? have a contributing.md file?
- [ ] what are your meetings like? are you uploading to youtube? 
- [ ] what programs do you participate in for new contributors?
- [ ] what programs are you participating in to grow contributors throughout the contributor ladder?
- [ ] what areas does the group need the most help with?
- [ ] whats the average open days of a PR and Issue in your group? / what metrics does your group care about?
- [ ] how can steering help?

**Comment by timothysc** [comment]
WRT working groups, they should be reporting to their SIGs and SIGs should readout to steering, the onus should be on them.  

WRT sig format, I think we should write a policy doc.  I like most of the suggestions listed above.

**Comment by parispittman** [comment]
note for later:
governance.md calls out chairs to communicate and coordinate with steering, just ambiguous:
> SIG chairs are intended to be organizers and facilitators, responsible for the operation of the SIG and for communication and coordination with the other SIGs, the Steering Committee, and the broader community.

**Comment by cblecker** [comment]
> WRT working groups, they should be reporting to their SIGs and SIGs should readout to steering, the onus should be on them.

As we are trying to also assess the activity of SIGs themselves, and WGs can have multiple sponsoring SIGs, I think the process as a whole would be sped up if steering assessed all the governed groups (SIGs, WGs, UGs, Committees, etc) directly. You're not wrong that it's the SIGs responsibility, but I just think it'll take longer for us to get the information we're looking for in our governance oversight role. 

There could probably also be a question to the SIGs "Have you had regular reports from all the WGs you sponsor?"

**Comment by spiffxp** [comment]
/unassign @timothysc 
/assign @nikhita @cblecker @lachie83 
Discussed during steering today, will work to have some open questions resolved by next steering committee meeting.  Possible breakout session to discuss in the interim.

**Comment by parispittman** [comment]
breakout session scheduled for 4/27

**Comment by parispittman** [comment]
/close

This is done. :)
I’ve opened #164 to track the next stage of roll out 


https://github.com/kubernetes/community/pull/4687
